packages/openapi-code-generator/src/core/openapi-types-normalized.ts

```diff
@@ -70,6 +74,8 @@ export type IRModel =
   | IRModelBoolean
   | IRModelObject
   | IRModelArray
+  | IRModelAny
```

Comment:
I'm not sure how well this will generalize outside of `typescript`, but we'll give it a go for now